src/dialog.c

```diff
@@ -103,6 +103,24 @@ void dialog_update(Dialog *d) {
 
 		fapproach_asymptotic_p(&a->focus, a->target_focus, 0.12, 1e-3);
 	}
+
+	if (d->title.active) {
```

Comment:
should this be done with tasks? @Akaricchi 
like common_easing_animate!

Reply:
It can be, but I don't really know if it should be. The logic here isn't particularly convoluted without tasks, so I don't have a strong opinion here. Replacing the entirety of `dialog_update` with a task might be good I suppose, so that stage.c wouldn't have to call it explicitly.

Reply:
By the way, extraneous space after `if` on this line.